Contributing/Contributing-guidelines.md

```diff
@@ -3,6 +3,64 @@ Please add a new issue to the [repository](https://github.com/golemfactory/golem
 
 If you add information about an error, please append as much information as possible, try to add screenshots and attach your logs to the issue. You also can send your detailed comments to [contact@golem.network](mailto:contact@golem.network).
 
+The best way to do that is create an issue on Github:
+Use one of our templates:
+Put as many information as you can.
+
+## Description
```

Comment:
I would link to the template file(s) in the main repo instead of pasting the contents here.

when there are updates they need to be changed only in one place